orb_models/forcefield/pretrained.py

```diff
@@ -174,12 +134,17 @@ def orb_v2(
     return model
 
 
-def orb_d3_v1(
-    weights_path: str = "https://storage.googleapis.com/orbitalmaterials-public-models/forcefields/orb-d3-v1-20240902.ckpt",
+def orb_d3_v2(
+    weights_path: str = "https://storage.googleapis.com/orbitalmaterials-public-models/forcefields/orb-d3-v2-20241011.ckpt",  # noqa: E501
     device: Union[torch.device, str] = None,
 ):
-    """ORB v1 with D3 corrections."""
-    base = get_base()
+    """Load ORB D3 v2."""
+    base = get_base(
```

Comment:
For extra clarity, it would  be good to change the defaults in `get_base` (we could then skip passing these args; your call)

Reply:
Similarly, might be nice to change the defaults in gns.py and graph_regressor.py.

It's not necessary, just makes it easier to see what v2 uses when browsing the code.